### PR TITLE
Optimization for pangram string

### PR DESCRIPTION
**Pull request by shellhub**
### **Describe your change:**

* [x] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: strings/check_pangram.py [hunk 1]
```diff
@@ -24,7 +32,50 @@ def check_pangram(
...
+            "check_pangram()", setup="from __main__ import check_pangram", number=1000
+        )
+    )
+    print(
+        timeit.timeit(
+            "check_pangram_faster()",
+            setup="from __main__ import check_pangram_faster",
+            number=1000,
+        )
+    )
```
Comment (cclauss) [inline strings/check_pangram.py:74]:
```suggestion
    setup = "from __main__ import check_pangram, check_pangram_faster"
    print(timeit.timeit("check_pangram()", setup=setup))
    print(timeit.timeit("check_pangram_faster()", setup=setup))
```

File: strings/check_pangram.py [hunk 2]
```diff
@@ -24,7 +32,50 @@ def check_pangram(
...
+        if char.islower():
+            flag[ord(char) - ord("a")] = True
+    return all(flag)
+
+
+def benchmark() -> None:
+    """
+    Benchmark code comparing different version.
+    """
+    import timeit
```
Comment (cclauss) [inline strings/check_pangram.py:61]:
```suggestion
    from timeit import timeit
```

**Comment by shellhub** [comment]
Thanks :) @cclauss

**Comment by shellhub** [comment]
@cclauss Please reiview. :)

**Review by cclauss — APPROVED** [review]
LGTM

**Pull request merged**